pkg/printers/internalversion/printers.go

```diff
@@ -1328,25 +1326,42 @@ func printEvent(obj *api.Event, options printers.PrintOptions) ([]metav1beta1.Ta
 		Object: runtime.RawExtension{Object: obj},
 	}
 	// While watching event, we should print absolute time.
-	var FirstTimestamp, LastTimestamp string
+	var firstTimestamp, lastTimestamp string
 	if options.AbsoluteTimestamps {
-		FirstTimestamp = obj.FirstTimestamp.String()
-		LastTimestamp = obj.LastTimestamp.String()
+		firstTimestamp = obj.FirstTimestamp.String()
+		lastTimestamp = obj.LastTimestamp.String()
+	} else {
+		firstTimestamp = translateTimestampSince(obj.FirstTimestamp)
+		lastTimestamp = translateTimestampSince(obj.LastTimestamp)
+	}
+	if options.Wide {
+		row.Cells = append(row.Cells,
+			lastTimestamp,
+			obj.Type,
+			obj.Reason,
+			obj.InvolvedObject.Kind,
+			formatEventSource(obj.Source),
+			strings.TrimSpace(obj.Message),
+			obj.InvolvedObject.FieldPath,
+			firstTimestamp,
+			int64(obj.Count),
+			obj.Name,
+		)
 	} else {
-		FirstTimestamp = translateTimestampSince(obj.FirstTimestamp)
-		LastTimestamp = translateTimestampSince(obj.LastTimestamp)
+		row.Cells = append(row.Cells,
+			lastTimestamp,
+			obj.Type,
+			obj.Reason,
+			obj.InvolvedObject.Kind,
+			strings.TrimSpace(obj.Message),
+		)
 	}
-	row.Cells = append(row.Cells, LastTimestamp, FirstTimestamp,
-		int64(obj.Count), obj.Name, obj.InvolvedObject.Kind,
-		obj.InvolvedObject.FieldPath, obj.Type, obj.Reason,
-		formatEventSource(obj.Source), obj.Message)
 
 	return []metav1beta1.TableRow{row}, nil
 }
 
 // Sorts and prints the EventList in a human-friendly format.
 func printEventList(list *api.EventList, options printers.PrintOptions) ([]metav1beta1.TableRow, error) {
-	sort.Sort(events.SortableEvents(list.Items))
```

Comment:
Not sure where sorting is being done now, are they sorted in the `get` command now?

Reply:
The apiserver returns in natural order.  To sort events by timestamp, you have to ask for it - sort doesn't work at this level because chunking would result in the objects being sorted.

We might in the future add a feature that allows a resource to define its default sort on the client side, but we'd need to think more about it.